source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidRealTimeMultiplayerClient.cs

```diff
@@ -11,16 +11,36 @@ namespace GooglePlayGames.Android
 
     internal class AndroidRealTimeMultiplayerClient : IRealTimeMultiplayerClient
     {
+        private readonly object mSessionLock = new object();
+
+        private const float InitialPercentComplete = 20.0F;
+
+        private static readonly int ROOM_VARIANT_DEFAULT = -1;
+        private static readonly int ROOM_STATUS_INVITING = 0;
```

Comment:
ROOM_STATUS* to enum?

Reply:
Converted and referenced to the Java constants with comments. I have an extra status here to handle null cases which is specified with a comment.